src/System.Private.CoreLib/src/System/Threading/LowLevelSpinWaiter.cs

```diff
@@ -14,12 +14,12 @@ namespace System.Threading
     /// 
     /// Used by the wait subsystem on Unix, so this class cannot have any dependencies on the wait subsystem.
     /// </summary>
-    internal struct FirstLevelSpinWaiter
+    internal struct LowLevelSpinWaiter
     {
         // TODO: Tune these values
         private const int SpinCount = 8;
-        private const int SpinYieldThreshold = 4;
-        private const int SpinSleep0Threshold = 6;
+        private const int SpinYieldThreshold = 6;
+        private const int SpinSleep0Threshold = 4;
```

Comment:
Can remove `SpinYieldThreshold` and replace all of its uses with `SpinSleep0Threshold` (and keep `SpinSleep0Threshold = 4` as you have now)

Reply:
I could do it if the uniprocessor case is refactored differently (see below). I wanted to keep the same number of yields as before this PR.

Reply:
Sleep(0) is a superset of Yield, so Yield is not necessary anymore and replacing it with Sleep0 with the previous (lower) threshold used for Yield would retain the same effective threshold as before this PR

Reply:
Agreed on that. I'm fine with replacing `SpinYieldThreshold` with `SpinSleep0Threshold` if you are.